src/main/java/org/jenkinsci/plugins/github/webhook/WebhookManager.java

```diff
@@ -291,6 +310,9 @@ public GHHook apply(GHRepository repo) {
     protected Predicate<GHHook> deleteWebhook() {
         return new Predicate<GHHook>() {
             public boolean apply(GHHook hook) {
+                if (hook ==null) {
```

Comment:
i can get null here instead of hook if only myself call this method with null. And this is impossible, because of hooks deleted only after filtering